apis/druid/v1alpha1/druid_types.go

```diff
@@ -288,9 +288,12 @@ type DruidNodeSpec struct {
 	// Optional
 	ReadinessProbe *v1.Probe `json:"readinessProbe,omitempty"`
 
-	// Optional: StartupProbe for nodeSpec
+	// Optional: StartupProbe for nodeSpec, this field is deprecated use `startUpProbe` instead
 	StartUpProbes *v1.Probe `json:"startUpProbes,omitempty"`
 
```

Comment:
This can be removed, ill track this in the next release. Might be a little incompatible but IMHO its not needed.